api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -86,4 +87,11 @@ message QuicProtocolOptions {
   // If not specified, no debug visitor will be attached to connections.
   // [#extension-category: envoy.quic.connection_debug_visitor]
   core.v3.TypedExtensionConfig connection_debug_visitor_config = 11;
+
+  // Configure a type of UDP cmsg to pass to listener filters via QuicReceivedPacket.
+  // Both level and type must be specified for cmsg to be saved.
+  // Cmsg may be truncated or omitted if expected size is not set.
```

Comment:
I wonder if this part of the comment true for CMSG headers in general (and then it should probably be part of the description of `SocketCmsgHeaders`. However, if it is only related to the QUIC related implementation, then I agree it should be specified here.

Reply:
As far as I know, this is specific to QUIC, but that's also because I simply don't know what other implementations might look like.